Automattic/wp-calypso
# Add FullStory Analytics Components

#### Changes proposed in this Pull Request

* Add new Component, `<FullStoryRecorder/>` to Calypso
   *  Triggers tacking for logged in, US users, when rendered ( and `record` property not set to false )
* Add new Component, `<AnalyticsSafeContainer/>` to protect sensitive user information
   * In addition to FullStory this should implement safety for any other new analytics packages used
   * Care should be user to make all areas that could need protecting are found and covered
 

#### Testing instructions
 1. Add `FullStoryRecorder` and `AnalyticsSafeContainer` to multiple pages of a flow ( see my example at #52648 )
 2. Run in development mode
 3. Verify recording the flow works in the FullStory dashboard
